src/MessageHandler/ActivityPub/Outbox/DeliverHandler.php

```diff
@@ -53,8 +59,21 @@ public function workWrapper(MessageInterface $message): void
             $this->doWork($message);
             $conn->commit();
         } catch (InvalidApPostException $e) {
+            if (400 <= $e->responseCode && 500 > $e->responseCode && 429 !== $e->responseCode) {
```

Comment:
```suggestion
            if (400 <= $e->responseCode && 500 => $e->responseCode && 429 !== $e->responseCode) {
```

Also equal to 500 errors right?

Reply:
nope 500 errors are server errors and therefore should subside on later tries.

Reply:
OK but 502, 503 & 504 (and more?) could potentiality also be some kind of server configuration issue (just like 500 error would give if something is wrong on the server itself).

Reply:
Ah yes, right! gotcha 🥟